utils/sdk.py

```diff
@@ -0,0 +1,27 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+import logging
+import os
+
+# 3p
+import simplejson as json
+
+# project
+
+
+log = logging.getLogger(__name__)
+
+
+def load_manifest(path):
```

Comment:
It does, `except Exception` was intentionally not very specific.... Geared towards `IOErrors` and `JSONDecodeError`.